models/cow_protocol/arbitrum/cow_protocol_arbitrum_schema.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+
+models:
+  - name: cow_protocol_arbitrum_solvers
+    meta:
+      blockchain: arbitrum
+      project: cow_protocol
+      contributors: olgafetisova
+    config:
+      tags: ['arbitrum_chain','cow_protocol','solver']
+    description: >
+      CoW Protocol solvers list on Arbitrum Chain
+
+  - name: cow_protocol_arbitrum_batches
```

Comment:
can you add unique column test to this spell? we should validate the unique key used in model config block is unique. while the merge helps us do this, it's good to include the test to be safe